src/main/java/org/folio/circulation/domain/RequestServiceUtility.java

```diff
@@ -115,6 +115,23 @@ private RequestServiceUtility() { }
     }
   }
 
+  static Result<RequestAndRelatedRecords> refuseWhenItemToBeMovedIsFromDifferentInstance(
+    RequestAndRelatedRecords requestAndRelatedRecords) {
+
+    Request request = requestAndRelatedRecords.getRequest();
+    Item item = request.getItem();
+    if (!Objects.equals(item.getInstanceId(), request.getInstanceId())) {
+      String message = "Request can be moved only to an item with the same instance ID";
```

Comment:
If we're using this value only once, does it make sense to make it a variable?

Reply:
Refactored